java/java.source.base/src/org/netbeans/modules/java/source/TreeShims.java

```diff
@@ -60,6 +63,8 @@ public class TreeShims {
 //    public static final String NULL_LITERAL = "NULL_LITERAL"; //NOI18N
 //    public static final String PARENTHESIZED_PATTERN = "PARENTHESIZED_PATTERN"; //NOI18N
 //    public static final String GUARDED_PATTERN = "GUARDED_PATTERN"; //NOI18N
+    public static final String DECONSTRUCTION_PATTERN = "DECONSTRUCTION_PATTERN";
```

Comment:
Please note that after https://github.com/apache/netbeans/pull/4467, it should be possible to refer to the deconstruction/record patterns directly, without the need for reflection and `TreeShims`.

Reply:
> Please note that after #4467, it should be possible to refer to the deconstruction/record patterns directly, without the need for reflection and `TreeShims`.

Hi, yes. Have noticed that. Will commit the latest code in some time